src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/Microsoft.Extensions.Hosting.WindowsServices.csproj

```diff
@@ -27,6 +29,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Hosting\src\Microsoft.Extensions.Hosting.csproj" />
+    <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Logging.Abstractions\src\Microsoft.Extensions.Logging.Abstractions.csproj" />
```

Comment:
What code in WindowsServiceLifetime.cs caused the need to add this new project reference?

Reply:
Oh, this is the transitive dependency problem you described here, right? https://github.com/dotnet/runtime/pull/85656#issuecomment-1531890836

Reply:
Correct.  It's a side-effect of incremental servicing and our package testing caught it.  Good reason to pay attention to those failures 👍